components/commander/common/features.cc

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/commander/common/features.h"
+
+#include "base/feature_list.h"
+
+namespace features {
+
+BASE_FEATURE(kBraveCommander,
+             "BraveCommander",
+             base::FEATURE_DISABLED_BY_DEFAULT);
+}
+
+namespace commander {
+
+bool CommanderEnabled() {
```

Comment:
`IsCommanderEnabled`, but this pattern is generally only used when there are more conditions than just a single buildflag so I would probably remove this